ucp/continuous_ucx_progress.py

```diff
@@ -7,21 +7,21 @@
 
 
 class ProgressTask(object):
-    def __init__(self, ctx, event_loop):
-        """Creates a task that keeps calling ctx.worker.progress()
+    def __init__(self, worker, event_loop):
+        """Creates a task that keeps calling worker.progress()
 
         Notice, class and created task is carefull not to hold a
-        reference to `ctx` so that a danling progress task will
-        not prevent `ctx` to be garbage collected.
+        reference to `worker` so that a danling progress task will
+        not prevent `worker` to be garbage collected.
 
         Parameters
         ----------
-        ctx: ApplicationContext
-            The application context to progress
+        worker: UCXWorker
+            The UCX worker context to progress
```

Comment:
Is this breaking API? If so, are there consequences for downstream users (like Distributed or cuML)?

Reply:
This is not part of the public API and AFAIK nobody are using it downstream.